src/data/int/basic.lean

```diff
@@ -140,6 +140,20 @@ not_congr coe_nat_eq_zero
 
 @[simp] lemma coe_nat_nonneg (n : ℕ) : 0 ≤ (n : ℤ) := coe_nat_le.2 (nat.zero_le _)
 
+@[simp] lemma le_coe_nat_sub (m n : ℕ) :
```

Comment:
My instinct is that this is not a great simp lemma. It seems like a very uncommon pattern keyed on a very common head symbol. @eric-wieser do you agree?

Reply:
Ok. What would you say in general qualifies as good for simp? I was thinking if there were any computations involving sums of coercions etc. this would be nice.